docs/sources/visualizations/canvas.md

```diff
@@ -0,0 +1,19 @@
+---
+aliases:
+  - /docs/grafana/latest/features/panels/canvas/
+  - /docs/grafana/latest/visualizations/canvas/
+description: Canvas visualization documentation
+keywords:
+  - grafana
+  - canvas
+  - panel
+  - documentation
+title: Canvas
+weight: 600
+---
+
+# Canvas
+
+The Canvas panel visualization allows you to ...
+
+TODO: add gif from what's new
```

Comment:
Are GIFs supported? I wasn't able to get them to work previously.

Reply:
GIFs are supported in the documentation, but either static screenshots or video (even silent video) are preferred.

Reply:
It appears that they are based on gif for [this section](https://grafana.com/docs/grafana/next/whatsnew/whats-new-in-v9-2/#panel-help-menu) in current 9.2 what's new docs